LoopKit/RemoteDataService.swift

```diff
@@ -84,14 +84,38 @@ public extension RemoteDataService {
 
     var carbDataLimit: Int? { return nil }
 
+    func uploadCarbData(deleted: [DeletedCarbEntry], stored: [StoredCarbEntry], completion: @escaping (_ result: Result<Bool, Error>) -> Void) {
+        completion(.success(false))
```

Comment:
[nit] Since this is just a placeholder, it's fine, but I guess I expected something more along the lines of:
```
completion(.failed(.unimplemented))
```
...but it's a nit.

Reply:
Actually, I want the default to be `.success` with no data uploaded (that's the `false`). The default for any remote data service should be to successfully ignore the data to be uploaded, particularly as we add more data types that are specific to Tidepool Loop and not at all supported by Nightscout.